### PR TITLE
Release version `7.6-preview.1` for Key Vault data plane

### PR DESCRIPTION
**Pull request by vcolin7**

### CODE REVIEW

**Comment by heaths** [comment]
The ModelValidation issue is a known issue where it doesn't like "/" as a parameter value, which the service accepts as the scope.

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json [hunk 1]
```diff
@@ -82,6 +82,62 @@
...
+        "responses": {
+          "202": {
+            "description": "Queued validating pre full backup operation",
+            "headers": {
+              "Retry-After": {
+                "description": "The recommended number of seconds to wait before calling the URI specified in Azure-AsyncOperation.",
+                "type": "integer",
+                "format": "int64"
+              },
+              "Azure-AsyncOperation": {
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json:118]: This is not the correct method for long-running operations. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#lro-returns-operation-location and https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs.

We could allow this for a prerelease, but it'll need to be fixed for GA. Please inform the certs team.

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json [hunk 2]
```diff
@@ -122,6 +178,62 @@
...
     },
+    "/prerestore": {
+      "put": {
+        "tags": [
+          "PreFullRestore"
+        ],
+        "operationId": "PreFullRestoreOperation",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json:189]: Same here - and will also impact the `final-state-via` above.

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json [hunk 3]
```diff
@@ -281,6 +393,18 @@
...
   "definitions": {
+    "PreRestoreOperationParameters": {
+      "properties": {
+        "sasTokenParameters": {
+          "$ref": "#/definitions/SASTokenParameter"
+        },
+        "folderToRestore": {
+          "type": "string",
+          "description": "The Folder name of the blob where the previous successful full backup was stored"
+        }
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json:404]: Shouldn't either or both of these be required?

File: specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json [hunk 4]
```diff
@@ -363,6 +487,24 @@
         "storageResourceUri"
       ]
     },
+    "PreBackupOperationParameters": {
+      "properties": {
+        "storageResourceUri": {
+          "type": "string",
+          "description": "Azure Blob storage container Uri"
+        },
```
Comment (heaths) [inline specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json:495]: Shouldn't this at least be required?

**Review by heaths — APPROVED** [review]
/cc @jlichwa for some comments I transferred from #28400. I think those should be addressed by at least GA, if not before 7.6-preview.1 ships i.e., the async-operation one. That's not how we recommend doing LROs. See https://aka.ms/azapi/guidelines.

**Comment by jlichwa** [comment]
> /cc @jlichwa for some comments I transferred from #28400. I think those should be addressed by at least GA, if not before 7.6-preview.1 ships i.e., the async-operation one. That's not how we recommend doing LROs. See https://aka.ms/azapi/guidelines.

@riteshchitlangia1 can you take a look on those issues?

**Comment by heaths** [comment]
> > /cc @jlichwa for some comments I transferred from #28400. I think those should be addressed by at least GA, if not before 7.6-preview.1 ships i.e., the async-operation one. That's not how we recommend doing LROs. See https://aka.ms/azapi/guidelines.
> 
> @riteshchitlangia1 can you take a look on those issues?

Just let us know whether you plan to address those now or later, and we can react on this PR as appropriate.

**Review by mikekistler — APPROVED** [review]
Looks good for preview. 👍

**Comment by riteshchitlangia1** [comment]
> > > /cc @jlichwa for some comments I transferred from #28400. I think those should be addressed by at least GA, if not before 7.6-preview.1 ships i.e., the async-operation one. That's not how we recommend doing LROs. See https://aka.ms/azapi/guidelines.
> > 
> > 
> > @riteshchitlangia1 can you take a look on those issues?
> 
> Just let us know whether you plan to address those now or later, and we can react on this PR as appropriate.

@heaths , we can go ahead with current release if it is not a blocker and @vasanthrajams who originally added these changes can look at the comments when he is back next week.

**Pull request merged**